src/box/tuple_dictionary.c

```diff
@@ -175,6 +177,32 @@ tuple_dictionary_new(const struct field_def *fields, uint32_t field_count)
 	return NULL;
 }
 
+uint32_t
+tuple_dictionary_hash_process(struct tuple_dictionary *dict,
```

Comment:
Dict argument here and in the other function can be made `const`.

Reply:
Sure. Fixed here and in tuple_dictionary_cmp.